index.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 var isFinite = require('is-finite');
 
-module.exports = function (numbers) {
+module.exports = function (numbers, opts) {
 	if (!Array.isArray(numbers)) {
 		throw new TypeError('Expected an array');
 	}
 
 	var ticks = ['▁', '▂', '▃', '▄', '▅', '▆', '▇', '█'];
 	var finiteNumbers = numbers.filter(isFinite);
-	var min = Math.min.apply(null, finiteNumbers);
-	var max = Math.max.apply(null, finiteNumbers);
+  opts = opts || {};
```

Comment:
Quick question, if you don't mind. Is my thinking on this correct? 
I don't have to use `var opts = opts || {};`. We don't risk elevating to global scope here because `opts` is in the function declaration and so `var opts` is assumed implicitly?